src/clients/zodiac/abis/zodiac.json

```diff
@@ -0,0 +1,389 @@
+[
+  {
```

Comment:
Can we use string format here? It's much more compact, here is a function to turn this to string format  https://github.com/snapshot-labs/snapshot.js/blob/master/test/abi.ts